datetime.go

```diff
@@ -60,3 +60,34 @@ func snowflakeFormatToGoFormat(sfFormat string) (string, error) {
 	}
 	return res, nil
 }
+
+func dateTimeFormatByType(dateTimeType string, params map[string]*string) (string, error) {
```

Comment:
test for that?

Reply:
It is tested via integration test. Do you see a value in unit testing?